src/main/java/org/madhuri/app/message/model/StockResponse.java

```diff
@@ -0,0 +1,41 @@
+package org.madhuri.app.message.model;
+
+import java.util.List;
+
+import jakarta.json.bind.annotation.JsonbProperty;
+
+public class StockResponse {
+
+	@JsonbProperty("count")
+	private long count;
+	
+	@JsonbProperty("result")
+	private List<StockSymbol> result;
+	
+	public StockResponse()
```

Comment:
Do we need an empty constructor ?

Reply:
The default constructor is especially used during deserialization when converting JSON or data formats into Java objects.